### PR TITLE
Increase coverage to 86%

### PR DESCRIPTION
**Pull request by matiaspando**
Statements   : 86.83% ( 3053/3516 )
Branches     : 72.73% ( 984/1353 )
Functions    : 85.48% ( 530/620 )
Lines        : 87.94% ( 3019/3433 )

### CODE REVIEW

File: js/models/Wallet.js [hunk 1]
```diff
@@ -713,6 +713,10 @@ Wallet.prototype._onData = function(senderId, data, ts) {
 
   this.updateSyncedTimestamp(ts);
 
+  console.log('data.type ', data.type);
+  console.log('this.id ', this.id);
+  console.log('data.walletId ', data.walletId);
+
```
Comment (cmgustavo) [inline js/models/Wallet.js]: Please, remove the console.log

**Comment by matiaspando** [comment]
Statements   : 88.38% ( 3104/3512 )
Branches     : 74.65% ( 1010/1353 )
Functions    : 87.42% ( 542/620 )
Lines        : 89.41% ( 3066/3429 )

**Comment by matiaspando** [comment]
Statements   : 88.61% ( 3112/3512 )
Branches     : 75.24% ( 1018/1353 )
Functions    : 87.74% ( 544/620 )
Lines        : 89.56% ( 3071/3429 )

**Comment by cmgustavo** [comment]
ack. all test passed.

**Pull request merged**